pkg/generator/generator.go

```diff
@@ -169,6 +169,17 @@ func (g Generator) generateRawConfig(ctx context.Context, app string) (configmap
 	g.logMessage(ctx, "decrypted installation secret")
 
 	// 6.
+	secretContextFinal, err := applyPatch(
+		ctx,
+		[]byte(configmapContext),
+		[]byte(secretContext),
+	)
+	if err != nil {
+		return "", "", microerror.Mask(err)
+	}
+	g.logMessage(ctx, "merged config and secret values")
+
+	// 7.
```

Comment:
Now it doesn't match the function's godoc.

Reply:
updated